test/Create3Deployer.js

```diff
@@ -154,6 +154,25 @@ describe('Create3Deployer', () => {
       expect(await contract.decimals()).to.equal(decimals);
     });
 
+    it('should deploy with passing native value to the contact', async () => {
+      const key = 'a test key';
+      // Send 1 eth to address
+      const amount = ethers.utils.parseEther('1');
```

Comment:
```suggestion
      // Send eth to address
      const amount = ethers.utils.parseEther('0.00000001');
```
This is expensive when running on a live network